src/EnergyPlus/PlantPressureSystem.cc

```diff
@@ -493,25 +485,27 @@ void UpdatePressureDrop(EnergyPlusData &state, int const LoopNum)
 
         // Loop through all branches on this loop side
         LoopSidePressureDrop = 0.0;
-        NumBranches = size(state.dataPlnt->PlantLoop(LoopNum).LoopSide(LoopSideNum).Branch);
+        int NumBranches = size(state.dataPlnt->PlantLoop(LoopNum).LoopSide(LoopSideNum).Branch);
 
         // Split here based on a single branch loop or a splitter/mixer configuration
         if (NumBranches == 1) { // Just do the single branch
 
             //***SINGLE BRANCH***!
             BranchNum = 1;
-            DistributePressureOnBranch(state, LoopNum, LoopSideNum, BranchNum, BranchPressureDrop, FoundAPumpOnBranch);
-            LoopSidePressureDrop += BranchPressureDrop;
-            LoopPressureDrop += BranchPressureDrop;
+            Real64 BranchPressureDropValue = 0.0;
```

Comment:
BranchPressureDrop is also a function name which showed as a shadow outer function error after this scope change so I changed the variable name.

void BranchPressureDrop(EnergyPlusData &state,